examples/rdflib/03_parse_grouped.py

```diff
@@ -0,0 +1,23 @@
+from rdflib import Dataset, Graph
+
+# libraries to load example jelly stream data
+import gzip
+import urllib.request
+from typing import cast, IO
+
+from pyjelly.integrations.rdflib.parse import parse_jelly_grouped
+
+url = "https://w3id.org/riverbench/datasets/dbpedia-live/dev/files/jelly_10K.jelly.gz"
+
+# load, uncompress .gz file, and pass to jelly parser
+with (
+    urllib.request.urlopen(url) as resp,
+    cast(IO[bytes], gzip.GzipFile(fileobj=resp)) as jelly_stream,
+):
+    graphs = parse_jelly_grouped(
+        jelly_stream,
+        graph_factory=lambda: Graph(),
+        dataset_factory=lambda: Dataset(),
+    )
+    for graph in graphs:
+        print(f"Number of triples in graph: {len(graph)}")
```

Comment:
Could we zip the `graphs` generator with the index? `enumerate()` does this.

Then you can print `f"Graph {i} in the stream has {len(graph)} triples"`

Try to be as natural as you can be in these examples :) :)